tests/integration/test_render.py

```diff
@@ -33,7 +33,11 @@ class TestDragAndDropRender(BaseIntegrationTest):
     PAGE_ID = 'drag_and_drop_v2'
     ITEM_PROPERTIES = [{'text': '1'}, {'text': '2'}, {'text': 'X'}, ]
 
-    def load_scenario(self, item_background_color="", item_text_color=""):
+    def load_scenario(self, item_background_color="", item_text_color="", borders=False):
+        if borders:
+            json = "integration/data/test_data_a11y_borders.json"
+        else:
+            json = "integration/data/test_data_a11y.json"
```

Comment:
@agaylard To avoid having to touch multiple files when fixture data changes in the future, we should try and keep the number of fixture files minimal. Instead of introducing `test_data_a11y_borders.json`, you can parameterize the existing file (`test_data_a11y.json`) as follows:

1. Instead of providing an explicit value for `"displayBorders"` (and `"displayLabels"`, if you're going to add tests for that as well), use a placeholder:

   ```js
     "displayBorders": {display_borders_value},
     "displayLabels": {display_labels_value}
   ```

2. In `load_scenario`, replace the placeholder(s) based on the value of `display_zone_borders` (and `display_zone_labels`):

   ```python
   def load_scenario(
       self, item_background_color="", item_text_color="", display_zone_borders=False, display_zone_labels=False
   ):
       data = load_resource("integration/data/test_data_a11y.json")
       data.replace('{display_borders_value}', 'true' if zone_borders else 'false')
       data.replace('{display_labels_value}', 'true' if zone_labels else 'false')
       scenario_xml = """
           <vertical_demo>
               <drag-and-drop-v2 item_background_color='{item_background_color}'
                                 item_text_color='{item_text_color}'
                                 data='{data}' />
           </vertical_demo>
       """.format(
           item_background_color=item_background_color,
           item_text_color=item_text_color,
           data=data
       )
   ```